examples/pom.xml

```diff
@@ -63,6 +63,7 @@
         <module>karaf-soap-example</module>
         <module>karaf-websocket-example</module>
         <module>karaf-war-example</module>
+        <module>karaf-redis-example</module>
```

Comment:
Can you please sort by alphanumeric ? `redis` should be before `s*` examples.

Reply:
Solved.